libbeat/publisher/pipeline/controller.go

```diff
@@ -128,6 +137,8 @@ func (c *outputController) Set(outGrp outputs.Group) {
 		}
 	}
 
+	c.out = grp
```

Comment:
Not sure why we didn't not set this before.

Reply:
yep, I wasn't sure this is needed, but it looks like it is